build-repo.sh

```diff
@@ -17,78 +17,56 @@
 
 set -xe
 
-mkdir -p binaries
+# Aptly does not need sudo, as the jenkins user is in the aptly group
+# Moving files to a seperate path to make sure we only include what 
+# we want in the repos
 
-export PROVIDE_ONLY=true
-export SUDO_CMD=sudo
-export BASE_PATH="binaries/"
-export APTLY_ONLY="focal"
+BASE_PATH="$(pwd)/binaries/"
+mkdir $BASE_PATH || true
 
-# Aptly does not need sudo, as the jenkins user is in the aptly group
+push_aptly() {
+    local release="$1"
+    local path="$2"
+
+    if ! aptly -db-open-attempts=40 repo show $release ; then
+      aptly -db-open-attempts=40 repo create -distribution="$release" $release
+      aptly -db-open-attempts=40 publish repo $release filesystem:repo:main
+    fi
+    aptly -db-open-attempts=40 repo include -repo="$release" "$path"
+    aptly -db-open-attempts=40 publish update $release filesystem:repo:main
+}
 
 if [ -f multidist.buildinfo ]; then
-	echo "Doing multibuild"
-	MULTI_DIST=$(cat multidist.buildinfo)
+  echo "Doing multibuild"
+  MULTI_DIST=$(cat multidist.buildinfo)
   for t in multidist*.tar.gz ; do
     tar --overwrite -xvzf $t
   done
-	rm multidist*.tar.gz || true
-  export rootwp=$(pwd)
+  rm multidist*.tar.gz || true
+  rootwp=$(pwd)
 
-	for d in $MULTI_DIST ; do
-		echo "Repo-ing for $d"
-		export distribution="$d"
-    export release="$d"
-    export REPOS="$release"
-    export WORKSPACE="$rootwp/mbuild/$d"
+  for d in $MULTI_DIST ; do
+    echo "Repo-ing for $d"
+    release="$d"
+
+    WORKSPACE="$rootwp/mbuild/$d"
     cd "$WORKSPACE"
-    mkdir $BASE_PATH || true
+    
+    # Copy back for jenkins to archive
```

Comment:
Doesn't archiving happen before this script is run?

https://github.com/ubports/build-tools/blob/30e43786855be735e37fa73428c796287e70fc78/Jenkinsfile#L58-L59

Reply:
Huh it does indeed, i can remove that then :)